.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
```

Comment:
vcs.xml on my machine has lines that mention the github branch protection rules. I'm worried that committing a config file without those branch protection rules could remove that.

If we add this block between line 2 and 3 we should be okay.

``` 
  <component name="GithubSharedProjectSettings">
    <option name="branchProtectionPatterns">
      <list>
        <option value="master" />
      </list>
    </option>
  </component>